src/main/java/bio/terra/flight/dataset/create/CreateDatasetAuthzResource.java

```diff
@@ -3,6 +3,7 @@
 import bio.terra.controller.AuthenticatedUserRequest;
 import bio.terra.exception.InternalServerErrorException;
 import bio.terra.flight.study.create.CreateStudyAuthzResource;
+import bio.terra.pdao.bigquery.BigQueryContainerInfo;
```

Comment:
An observation - true in my code also - that bigquery and gcp specific things are being directly used in the flights. That breaks the encapsulation I had hoped we would have. Ideally, we would have a platform-independent layer that would hide this. I wrote this up in DR-262.

Probably not worth dealing with in Q2, but it does represent technical debt, I think.